Source/XUnitExtensions/Lit/OrCommand.cs

```diff
@@ -12,11 +12,13 @@ public class OrCommand : ILitCommand {
     }
 
     public (int, string, string) Execute(ITestOutputHelper? outputHelper, TextReader? inputReader, TextWriter? outputWriter, TextWriter? errorWriter) {
-      var (exitCode, output, error) = lhs.Execute(outputHelper, inputReader, outputWriter, errorWriter);
-      if (exitCode == 0) {
-        return (exitCode, output, error);
+      var (leftExitCode, leftOutput, leftError) = lhs.Execute(outputHelper, inputReader, outputWriter, errorWriter);
+      if (leftExitCode == 0) {
+        return (leftExitCode, leftOutput, leftError);
       }
-      return rhs.Execute(outputHelper, inputReader, outputWriter, errorWriter);
+
+      var (rightExitCode, rightOutput, rightError) = rhs.Execute(outputHelper, inputReader, outputWriter, errorWriter);
+      return (rightExitCode, leftOutput + rightOutput, leftError + rightError);
```

Comment:
Not blocking, but this might help tell where the first output/error starts and the second begins:

```suggestion
      return (rightExitCode, leftOutput + "\n" + rightOutput, leftError + "\n" + rightError);
```

Reply:
I had that exact thought, but decided against it since that's not what the actual `||` shell operator does. :)

It should ultimately be moot since all we've ever used `||` for historically has been `|| true`, and we now have the strictly better `!` operator for that.